plugins/node/opentelemetry-instrumentation-knex/test/index.test.ts

```diff
@@ -191,6 +191,61 @@ describe('Knex instrumentation', () => {
       );
     });
 
+    it('should catch better-sqlite3 errors', async () => {
```

Comment:
To constrain this PR I only added this change here. In a follow up PR after this merges, I want to refactor this file so that we can do a matrix test against multiple clients.